internal/version/version.go

```diff
@@ -0,0 +1,8 @@
+package version
+
+// GitCommit indicates which commit the rukpak binaries were built from
+var GitCommit string
```

Comment:
does the value get injected at build time? is it worth adding a comment?

Reply:
+1.

Also, I don't see this used anywhere else in the Go binary. Maybe we should take this out of this PR and create a separate issue to remind us to get version info into the binary and have a way to print it out.

IIRC the go compiler as of 1.18 or 1.19 now injects this git info automatically. We might not need to do it ourselves anymore.